.github/workflows/autotest.yml

```diff
@@ -5,7 +5,16 @@ on:
   - push
 
 jobs:
+  add_header_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
I doubt if we need to pull history for all branches/tags for updating headers

Reply:
I don't have much experience with the GitHub `actions/checkout`. 😅 

We at least need the entire history before the commit, since the `add_header` script uses it.

Reply:
Ah, I see. Correct me if I am wrong, this would mean that if I send a PR modifying, let's say, file a, b and c. The `add_header` script would still parse the entire rucio commit history and update all files, not just a,b and c?

Reply:
This got resolved, since the PR now only processes the changed files.